cfn/handler/request.go

```diff
@@ -38,6 +38,14 @@ type Request struct {
 	resourcePropertiesBody         []byte
 }
 
+func (r *Request) GetPreviousResourcePropertiesBody() []byte {
```

Comment:
Any reason for using the `GetPreviousResourcePropertiesBody` instead of just going with `PreviousResourcePropertiesBody`?

Reply:
The JVM has affected me more than I realized ;). I'll push up the shorter version.